src/java.base/share/classes/java/io/FilePermission.java

```diff
@@ -1029,9 +1029,9 @@ public String getActions() {
      *
      * <p>and you are calling the {@code implies} method with the FilePermission:
      *
-     * <pre>
-     *   "/tmp/scratch/foo", "read,write",
-     * </pre>
+     * {@snippet lang=java :
+     *     "/tmp/scratch/foo", "read,write",
```

Comment:
This doesn't look like a valid Java code. This is not even a well-formed expression. I'm not sure whether there are any standard requirements about this (the spec is vague here), but IntelliJ IDEA assumes that the Java snippet is a member, a statement, or an expression. It's likely that parse error will be displayed here in the IDE.

Reply:
Indeed, maybe a script was used to replace the <pre> </pre> tags, in which case it will checking for other non-code examples.